games/minetest/Portfile

```diff
@@ -44,27 +45,44 @@ post-extract {
     }
 }
 
-license                 LGPL-2.1+
+license                 LGPL
 categories              games
 platforms               darwin
-maintainers             nomaintainer
+maintainers             @Zweihorn
 description             open source infinite-world block sandbox game with survival and crafting
-long_description        ${description}.
-
-depends_build-append    path:bin/doxygen:doxygen \
-                        port:mesa
+long_description        - ${description} - \
+                         \
+                        Minetest (usually abbreviated as MT) is an open source voxel game \
+                        engine. Play one of our many games, mod a game to your liking, make your \
+                        own game, or play on a multiplayer server. \
+                         \
+                        Available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly \
+                        BSD, and Android. \
+                         \
+                        ** Minetest at <https://www.minetest.net> \
+                        * MT mods at <https://content.minetest.net/> * \
+                        * Database explained at <https://wiki.minetest.net/Database_backends> ** \
+                         \
+                        This ${name} port provides a cutting edge Minetest (stable).
+
+homepage                https://www.minetest.net
+
+depends_build-append    port:doxygen
```

Comment:
why is the "mesa" port not needed anymore?

Reply:
I may ask in return why was "mesa" needed in the past?

I presume this came from a sub-optimal legacy _irrlichtmt_ in the past. The new Portfile does not need this requirement as this is fulfilled by "irrlichtmt @1.9.0mt8_0" i.e. the current _irrlichtmt_ port as dependency to this port.

This was tested with a clean install and also was true with the prior _irrlichtmt_ port(s) already.

Reply:
OK - Apparently my above rationale was _wishful thinking_ or just wrong.

New rationale:
- "mesa" is not found by `port rdeps irrlichtmt | grep mesa`
- "mesa" is found by  `port rdeps minetest | grep mesa` even with the new Portfile missing "mesa".

Adding "mesa" to the new Portfile again. Cannot hurt and better safe than sorry.